itests/felix/src/test/java/org/apache/aries/rsa/itests/felix/rsa/TestRSAListener.java

```diff
@@ -57,7 +58,8 @@
          rsaCore(), //
          rsaProviderFastBin(), //
          echoTcpService(), //
-         configFastBinFreePort()
+         configFastBinFreePort(),
+                vmOption("-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005")
```

Comment:
Was this intended? Looks like it is opening a debug port.

Reply:
No, it wasn't and I reverted that change with commit d4e021c

Reply:
Ah ..looks good then. Seems I only looked into the first commit.